eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/configuration/steps/ListMergedContigsStepConfigurationTest.java

```diff
@@ -101,7 +101,7 @@ public void assertStepExecutesAndCompletes() {
     public void contigsWritten() throws Exception {
         assertStepExecutesAndCompletes();
 
-        assertEquals(new HashSet<>(Arrays.asList("CM001954.1", "CM001941.2")),
+        assertEquals(new HashSet<>(Arrays.asList("CM001954.1,CAE13", "CM001941.2,CAE1")),
```

Comment:
Has the logic to generate the merged contigs file been updated? I can't find it.

Reply:
The `ContigMongoReader` was not changed in this PR. It will be changed in another one to include in the header only the contigs present in the VCF file [EVA-1647](https://www.ebi.ac.uk/panda/jira/secure/RapidBoard.jspa?rapidView=148&view=planning&selectedIssue=EVA-1647&issueLimit=100)

The `ContigWriter` is used for both active and merged and it was modified to include the sequence name so it can be used when creating the VCF.